packages/client-sdk-nodejs/src/config/middleware/middleware.ts

```diff
@@ -34,6 +37,548 @@ export class MiddlewareMessage {
     }
     return 0;
   }
+
+  constructorName(): string {
+    return this._grpcMessage.constructor.name;
+  }
+
+  // Note: APIs that use streaming interceptors (e.g. GetBatch and SetBatch)
```

Comment:
there were no middleware messages shown for GetBatch or SetBatch when I tested locally